frontend/src/components/home/midsection/lower_mid.jsx

```diff
@@ -92,8 +92,8 @@ class LowerMid extends Component {
           </Col>
         </Row>
         <Row className="test justify-content-md-center">
-          {footerImages.map(i => (
-            <Col className="justify-content-md-center">
+          {footerImages.map((i, ii) => (
```

Comment:
same anti pattern comment as above.

Reply:
in this case the array is [static](https://github.com/the-difference-engine/ymim/blob/staging/frontend/src/components/home/midsection/lower_mid.jsx#L72) so not worried about reording or changes, this could be better written from a code standpoint to maybe avoid this